packages/date-parser/src/dateParserV3.test.js

```diff
@@ -590,3 +590,45 @@ describe('default Chrono parsers should work well despite system timezone', () =
     expect(res.asTimestamp().end).toEqual('2019-12-26T05:33:00.000+09:00');
   });
 });
+
+describe('rawResult', () => {
+  const defaultOpts = { parserVersion: PARSER_VERSION_3, output: 'date', timezoneRegion: 'Asia/Seoul' };
+
+  it('output date return rawResult', () => {
+    const res = parse('2019-12-01 - 2019-12-02', new Date('2019-12-26T02:14:05Z'), { ...defaultOpts });
```

Comment:
Should add a case with datetime input e.g. 2019-12-02 12:00:00 - 2012-12-03 23:59:59